### ISSUE TITLE
Hypha Results (Jan 2021)

### ISSUE THREAD

**Issue by danaklug** [opensourceantibiotics/Series-2-Diarylimidazoles#50]
We have received results from the Hypha screen of OSA_000821 against their CYP panel (more information [here]( https://www.hyphadiscovery.co.uk/wp-content/uploads/2019/11/PolyCYP-info-sheet-Nov-2019.pdf); raw data [here]( https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Metabolism/OSA821%20screening%20data%202020_12_11_PolyCYP_FMO_AO_Human_S9_screens_UCL-BT-A.xlsx); and notebook page [here]( https://au-mynotebook.labarchives.com/share/Dana%2520Klug/NDA0LjN8Njk2ODYvMzExL1RyZWVOb2RlLzI2OTkxOTYxOTR8MTAyNi4z)). 

**Summary**

The compound of interest (**OSA_000821**) is incubated with panel enzymes, which carry out metabolic reactions. The resulting mixture is analyzed for metabolites of interest. Output is mass, retention time, and TIC (I believe) of metabolites.

The major species detected are two monohydroxylated metabolites (+16 Da, 0.88 min and 0.95 min) produced by multiple CYPs and one glucuronide (+176 Da, 0.90 min) produced by human liver S9 enzymes.

**Follow-up and outstanding questions**

- The next step is to scale up, isolate, and identify the metabolites of interest and discussions regarding this are ongoing. 

- Can we get confirmation from Hypha that the data provided in the spreadsheet is indeed the TIC of each species?

**Comment by MFernflower** [comment]
@danaklug I'm guessing since the glucuronide will be difficult the next step would be to incubate a bunch of OSA_000821 with lots of PolyCYP 483 and try to determine the two sites of hydroxylation - @mattodd will hypha do this for us or will it be done in UCL

**Comment by mattodd** [comment]
Yes, the next steps for this very useful initial work is to check the conversion of the PolyCYP 843 oxidation of OSA_000821 and then scale it up, which Hypha have offered to do (probably in February). Then handover of the material to us here at UCL to purify the target metabolites and elucidate the structures with NMR. I'll post any developments received by email.

**Comment by MFernflower** [comment]
Thanks @mattodd See you friday!

**Comment by mattodd** [comment]
Update from Julia at Hypha by email Feb 15th.

"The dose escalation study is complete and Emily will be undertaking the scale-up so that, all being well, we will be able to send you the extract for purification week commencing 1st March.

We are dosing the substrate in the scale-up at 300mg/L which should give a higher absolute yield of the +16s, although the overall % conversion will be lower than that observed at a lower dose.

One of the +16s is very minor and Richard thinks this may not be related to any enzyme reaction, but the other +16 is definitely a metabolite of PolyCYP 483. Structure elucidation of the products should reveal all!"

Thank you Hypha colleagues, we're looking forward to receiving this and isolating the metabolite.

**Comment by mattodd** [comment]
Scale-up material has been shipped from Hypha and should arrive at UCL tomorrow.

**Comment by danaklug** [comment]
Hypha HPLC chromatograms have been [uploaded](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Metabolism/Hypha_OSA_000821_HPLC.docx) and attached to the notebook page above.

**Comment by edwintse** [comment]
UPDATE: Hypha metabolite purification has been completed. I've got ~1-2 mg (balance accuracy questionable...) but I've managed to get 1H, 13C, COSY, HSQC and HMBC on it and have confirmed the structure as hydroxylated on the southern methylene carbon.

**Comment by MFernflower** [comment]
@edwintse perhaps it might be worth taking some OSA_000821 and mixing it with MCPBA in DCM to see if you can abiotically make more of the hypa metabolite?

**Comment by MFernflower** [comment]
@edwintse @danaklug The paper referenced from above:

https://applbiolchem.springeropen.com/track/pdf/10.1007/s13765-018-0404-2.pdf

**Comment by edwintse** [comment]
Characterisation data for the isolated metabolite (1D and 2D NMRs and the mnova file) have been uploaded [here](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/tree/master/Metabolism).
